App/screens/CredentialOffer.tsx

```diff
@@ -78,31 +126,28 @@ const CredentialOffer: React.FC<Props> = ({ navigation, route }) => {
             text1: t('CredentialOffer.RejectingCredential'),
           })
           try {
-            await agent?.credentials.declineOffer(credentialId)
-            Toast.show({
-              type: 'success',
-              text1: t('CredentialOffer.SuccessfullyRejectedCredential'),
-            })
-            navigation.goBack()
-          } catch {
+            await agent.credentials.declineOffer(credential.id)
+          } catch (e: unknown) {
             Toast.show({
               type: 'error',
-              text1: t('Global.Failure'),
+              text1: (e as Error)?.message || t('Global.Failure'),
             })
           }
         },
       },
     ])
   }
 
+  const connection = getConnectionRecordFromCredential(credential.connectionId)
+
   return (
     <View style={styles.container}>
       <ModularView
-        title={parseSchema(credential?.metadata.schemaId)}
+        title={parseSchema(credential.metadata.get<Record<string, string>>(INDY_CREDENTIAL_KEY)?.['schema_id'])}
```

Comment:
If you get the latest version of AFJ you can do the following:

```suggestion
        title={parseSchema(credential.metadata.get<IndyCredentialMetadata>(INDY_CREDENTIAL_KEY)?.schemaId)}
```

Which is a bit more type-safe :)

Reply:
Getting a TS error using the new `IndyCredentialMetadata` type:

```
Type 'IndyCredentialMetadata' does not satisfy the constraint 'Record<string, unknown>'.
Index signature for type 'string' is missing in type 'IndyCredentialMetadata'.
```